api/registrations/serializers.py

```diff
@@ -139,6 +154,44 @@ def get_registration_url(self, obj):
     def get_absolute_url(self, obj):
         return self.get_registration_url(obj)
 
+    def create(self, validated_data):
+        auth = get_user_auth(self.context['request'])
+        draft = validated_data.pop('draft')
+        registration_choice = validated_data.pop('registration_choice', 'immediate')
+        embargo_lifted = validated_data.pop('lift_embargo', None)
+
+        self.all_required_questions_answered(draft)
```

Comment:
Really minor semantic change-- `all_required_questions_answered` to me indicates a method returning a boolean value, when in fact it always returns a falsey value. This may be clearer as `check_required_questions_answered`